website/docs/components/pills/pills.md

```diff
@@ -33,43 +23,46 @@ const App = PlaygroundGeneration(
       options: ['m', 'l'],
     });
 
-    const selected = empty({
-      key: 'selected',
-      defaultValue: 1,
+    const disabled = bool({
```

Comment:
Maybe we should move `disabled` property under props for addons? As if addons are more important here, and `disabled` prop is usually somewhere under the others.